# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gdmap/05-no-error-on-missing-settings.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Description: Avoid printing an error message when the settings file isn't found
Author: Stephen Kitt <steve@sk2.org>
Last-Update: 2011-03-13

--- gdmap-v1.3.1/src/preferences.c.vanilla	2025-06-28 11:16:50.204000000 +0200
+++ gdmap-v1.3.1/src/preferences.c	2025-06-28 11:18:10.832585451 +0200
@@ -657,7 +657,10 @@
 
     if (!g_key_file_load_from_file(file, filename, 0, &error))
     {
-        g_message("%s", error->message);
+        if (error->code != G_FILE_ERROR_NOENT) {
+          // The settings file is allowed to not exist
+           g_message("%s", error->message);
+        }
         g_error_free(error);
         return;
     }
